spec/models/book_spec.rb

```diff
@@ -29,4 +29,11 @@
     expect(book.creator_roles.last.creator).to eq(creator2)
     expect(book.creator_roles.last.role).to eq(role2)
   end
+
+  it 'requires dcnum to be unique' do
+    book.save
+    # create a new book object with the same dcnum as book: xyz
+    expect { described_class.create digital_cico_number: digital_cico_number }.to
+    raise_error ActiveRecord::RecordNotUnique
```

Comment:
CircleCI is failing because the line break between `to` and `raise_error` — you should be able to just put them on the same line.